packages/suite/src/hooks/wallet/useSendForm.ts

```diff
@@ -309,6 +318,22 @@ export const useSendForm = (props: UseSendFormProps): SendContextValues => {
         setDraftSaveRequest(false);
     }, [draftSaveRequest, setDraftSaveRequest, saveDraft, getValues, errors]);
 
+    useDidUpdate(() => {
+        const { outputs } = getValues();
+
+        const conversionToUse = areSatsUsed ? amountToSatoshi : formatAmount;
+
+        outputs.forEach((output, index) => {
+            if (!output.amount) {
```

Comment:
nitpicking, wouldn't be better this instead of return?
```
if (output.amount) {
   sendFormUtils.setAmount(...)
}
```

Reply:
I usually prefer to use ifs for figuring out when to abort the action, makes it a bit more predictable, since the desirable result is usually at the end and not inside of some other block. However, since there is just 1 condition it is not that important :)